src/OpenTK.Windowing.Common/OpenTK.Windowing.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
You need to multi-target the other projects too.

Reply:
Ah right, I forgot that I had changed more than one project.

Reply:
Fixed with latest commit.